code/API_definitions/carrier_billing_payment.yaml

```diff
@@ -0,0 +1,932 @@
+openapi: 3.0.3
+info:
+  description: Service Enabling Payments against OB Carrier Billing Systems
+  version: '0.5'
+  title: Carrier Billing Payment API

```

Comment:
Just suggest no adding "API" to the API name (as in others working groups)

Reply:
OK sure - I removed it.